docs/user-guide/multiple_sources.md

```diff
@@ -0,0 +1,62 @@
+# Multiple Sources for an Application
```

Comment:
Could you recall how to create an app with helm chart source on one side (one repo/path, helm registry), and values on the other side (another Git repo/folder/version/...)?

Reply:
I would add an example of how to create an application with the above example